package.json

```diff
@@ -1,94 +1,95 @@
 {
-  "name": "react-components",
```

Comment:
Can we revert the style changes on package.json to only have your relevant changes please?